src/xcode/ENA/ENA/Source/Services/HealthCertificate/__test__/MockHealthCertificateService.swift

```diff
@@ -0,0 +1,29 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import OpenCombine
```

Comment:
`Never` comes with Foundation right?

Reply:
Seems like its nod needed to import Foundation here :)